yorkie/backend/cache.go

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2021 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * reference from the Kubernetes repository:
+ * https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/cache/lruexpirecache.go
+ */
+
+package backend
+
+import (
+	"container/list"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/yorkie-team/yorkie/pkg/types"
```

Comment:
How about removing the AuthWebhookResponse dependency and moving `cache.go` to `pkg/cache`?

Reply:
This decision was made because it is only used in AuthWebhook. But I agree with you because this code is likely to be used for utility. It's corrected!